### DISCUSSION TITLE
Add ESP32-C3 support to Tasmota IR firmware

### DISCUSSION THREAD

**Discussion by dado991** [arendst/Tasmota#17846] [Q&A] [answered]
Hi guys,
i've replaced a WBR3 with an ESP32-C3-12F inside an IR Bluster, do you think is possibile to add support for the C3 model of the ESP32 to the TASMOTA-IR firmare variant?

I tryied to compile by myself, but i have never used or compiled with platform.io.

Do you think there are same issue about that?

Thank You!

**Comment by Jason2866** [comment, marked as answer]
This should work. Add to your `platformio_tasmota_cenv.ini`

```
[env:tasmota32c3-ir]
extends = env:tasmota32c3
build_flags = ${env:tasmota32_base.build_flags} -DUSE_IR_REMOTE_FULL -DFIRMWARE_IR
```

**Reply by dado991** [reply]
Hi Jason,
i have compiled following this guide: https://tasmota.github.io/docs/Gitpod/#compile-your-firmware
adding your code at the bottom of `platformio_tasmota_cenv.ini`.
Start compiling with: `platformio run -e tasmota32c3-ir`

**And... It Worked!!! Thank You!** :)

Here the firmware running:
![immagine](https://user-images.githubusercontent.com/426608/216176746-3c0f9c45-4632-44d8-b9d2-143d08145178.png)

**Reply by kazeko123** [reply]
> Hi Jason, i have compiled following this guide: https://tasmota.github.io/docs/Gitpod/#compile-your-firmware adding your code at the bottom of `platformio_tasmota_cenv.ini`. Start compiling with: `platformio run -e tasmota32c3-ir`
> 
> **And... It Worked!!! Thank You!** :)
> 
> Here the firmware running: ![immagine](https://user-images.githubusercontent.com/426608/216176746-3c0f9c45-4632-44d8-b9d2-143d08145178.png)

Hello, my friend. I don't know much about programming. Could you please share the Tasmota IR firmware you are using on this ESP32-C3?
